packages/atomic/cypress/integration/facets/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -595,10 +595,14 @@ describe('Numeric Facet V1 Test Suites', () => {
         describe('with no facet results', () => {
           const baseSetup = () =>
             visibilitySetup().withCustomResponse((r) => {
-              const idx = r.facets.findIndex(
-                (facet) => facet.field === numericFacetField
-              );
-              r.facets[idx].values = [];
+              r.facets.forEach((facet, i) => {
```

Comment:
Test change needed since the internal logic is different. We need to "find" the new facet with the generated random ID, and then make only that one return no results.